core/strings/strings.odin

```diff
@@ -1424,12 +1426,29 @@ Output:
 
 */
 index_byte :: proc(s: string, c: byte) -> (res: int) {
-	for i := 0; i < len(s); i += 1 {
-		if s[i] == c {
-			return i
+	_index_byte :: #force_inline proc(s: string, c: byte) -> int {
+		for i := 0; i < len(s); i += 1 {
```

Comment:
'

Reply:
There's a fundamental difference when using the `for x in y` construct over a string, which is that it will iterate on a rune-by-rune basis, so if this review is suggesting that, then that would be slower than a per-byte iteration.